tests/Drupal/bad/bad.php

```diff
@@ -868,4 +868,12 @@ function test30(TestType $a = NULL) {
   echo "Hello";
 }
 
+/**
+ * Implements hook_node_view().
+ */
+#[Hook('hook_node_view')]
+function bad_node_view() {
```

Comment:
please also add a test for `hook_info` that is passes in the new sniff test file.